src/lecturer/components/pickQuizView/PickQuizList.tsx

```diff
@@ -0,0 +1,190 @@
+import React from "react";
+import {
+    Card,
+    CardHeader,
+    Checkbox,
+    Divider,
+    InputAdornment,
+    List,
+    ListItem,
+    ListItemIcon,
+    ListItemText,
+    TextField,
+} from "@material-ui/core";
+import { green } from "@material-ui/core/colors";
+import { makeStyles, useTheme } from "@material-ui/core/styles";
+import { Search } from "@material-ui/icons";
+interface QuizListViewProps {
+    title: React.ReactNode;
+    items: number[];
+    isQuiz: boolean;
+    numberOfChecked: (items: number[]) => number;
+    checked: number[];
+    questions: Question[];
+    handles: any;
```

Comment:
probably should be an array of functions type or better an object

Reply:
![image](https://user-images.githubusercontent.com/26445545/117540681-8dce3980-b010-11eb-8ea5-9e89d9d2dd98.png)
Is this better?